selfapi.js

```diff
@@ -395,7 +420,7 @@ API.prototype = {
         '<h1 id="' + getAnchor(this.title) + '">' + this.title + '</h1>\n';
     }
     if (this.description) {
-      html += '<p>' + this.description + '</p>\n';
+      html += '<p>' + this.description.replace(/\n/g, '<br/>') + '</p>\n';
```

Comment:
Nit: `<br>` is a [void element](https://html.spec.whatwg.org/#void-elements), which means it cannot have content or an end tag. Hence, the `/` character is useless, and ignored by web browsers (it's basically a comment, and adds a useless byte to the downloaded result). Please remove it.